### ISSUE TITLE
[OJS] Check/correct date formatting for Latest column in reviewer assignment

### ISSUE THREAD

**Issue by willinsky** [pkp/pkp-lib#2907]
Add Reviewer (OJS 3) does not appear to include year for Latest (review), which is helpful in establishing how long ago reviewer was active.

**Comment by asmecher** [comment]
@willinsky, I think there's something missing from the end of this. Also, do you intend to expose the names of the reviewers?

**Comment by willinsky** [comment]
Yes, I left the "also..." and the "and" hanging, after I decided that a further idea I had (for being able to delete reviewers) was better set out as a feature request filed elsewhere. I deleted the screenshot if that is what you meant by exposing the reviewers' names, as I can see it is not needed to make my point. However, I would note that the reviewer's identities are not something kept confidential (except in relation to their review of a particular work), and the names are often published to acknowledge their contribution. With Publons taking that to the next level.
